client/components/suggestions/index.jsx

```diff
@@ -158,7 +160,8 @@ class Suggestions extends React.Component {
 	 * @return {Object}          filtered taxonomy:[ terms ] object
 	 */
 	narrowDownAndSort = ( input, showAll = '' ) => {
-		const [ taxonomy, filter ] = input.split( ':' );
+		const termsTable = mapValues( this.props.terms, term => Object.keys( term ) );
```

Comment:
Can this be
`const termsTable = mapValues( this.props.terms, Object.keys );`
?